galaxy.yml

```diff
@@ -26,8 +26,8 @@ description: >
 
 # Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
 # accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
-license:
-- Unlicense
+#license:
+#- Unlicense
```

Comment:
Isn't this needed?

Reply:
@shawndwells either we specify it as `license` or `license_file`.... because it is specified as `license_files`, it is not needed.